helpers.lua

```diff
@@ -27,10 +27,8 @@ end
 local non_dialogue = {
     '^%b()$',
     '^（.-）$',
-    '^♬～$',
-    '^♬〜$',
-    '^♪～$',
-    '^～♪$',
+    '^-?%[.-%]$',
+    '^[♪♬～〜]+$',
```

Comment:
I've combined ```
    '^♬～$',
    '^♬〜$',
    '^♪～$',
    '^～♪$',
``` into this single line here so it'll match any line that has only these symbols (in any combination): `♪♬～〜`